dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java

```diff
@@ -163,15 +166,22 @@ public Class<? extends Enum<? extends ConnectorProperty>> getConnectorProperties
     return SnowflakeLogConnectorProperties.class;
   }
 
-  @ForOverride
-  protected String newQueryFormat(@Nonnull ConnectorArguments arguments)
+  private String newQueryFormat(@Nonnull ConnectorArguments arguments)
       throws MetadataDumperUsageException {
     // Docref: https://docs.snowflake.net/manuals/sql-reference/functions/query_history.html
     // Per the docref, Snowflake only retains/returns seven trailing days of logs.
-    return createQueryFromInformationSchema(arguments);
+    switch (inputSource) {
+      case USAGE_THEN_SCHEMA:
+        throw new IllegalArgumentException("Unsupported input source for Snowflake logs.");
+      case USAGE_ONLY:
```

Comment:
nit: I think the current naming is confusing because it does not reflect the snowflake table name.  `SCHEMA_ONLY` may suggest that only some kind of schema will be extracted but it actually changes source of query logs to `INFORMATION_SCHEMA` table.

Reply:
Done, I renamed the enum values and added Javadoc to clarify. New names include the word `source` to distinguish "schema" (information schema) as data source from schema as type of data.